.github/workflows/build.yaml

```diff
@@ -56,10 +56,10 @@ jobs:
             PATH: snow.zip
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python 3.10
+      - name: Set up Python 3.8
```

Comment:
Theoretically snowflake-python-connector supports 3.11 so we should be able to bump our version

Reply:
I will add new version in next PR